<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>300</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(161, 161, 161);</string>
  </property>
  <widget class="QLabel" name="IDNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:10pt;
font: 75 10pt &quot;Yu Gothic UI&quot;;</string>
   </property>
   <property name="text">
    <string>ID Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="accessButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(185, 185, 185);
font: 75 10pt &quot;Yu Gothic UI&quot;;</string>
   </property>
   <property name="text">
    <string>Access</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDTextBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="in_radio">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Yu Gothic UI&quot;;</string>
   </property>
   <property name="text">
    <string>In</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="out_radio">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Yu Gothic UI&quot;;</string>
   </property>
   <property name="text">
    <string>Out</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="admin_radio">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;Yu Gothic UI&quot;;</string>
   </property>
   <property name="text">
    <string>Admin</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>129</y>
     <width>491</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Messages</string>
   </property>
   <widget class="QLabel" name="redLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="greenLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 255, 0);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
